lc0/src/engine.cc

```diff
@@ -37,6 +37,8 @@ const char* kWeightsStr = "Network weights file path";
 const char* kNnBackendStr = "NN backend to use";
 const char* kNnBackendOptionsStr = "NN backend parameters";
 const char* kSlowMoverStr = "Scale thinking time";
+const char* kMoveOverheadStr = "Move Overhead";
```

Comment:
Maybe a bit descriptive name, so that both help and Uci parameter is more clear. Maybe "Move time overhead in milliseconds"?

It's fine to leave flag as it is.

Reply:
Move Overhead is the standard for all UCI engines